creator-node/src/routes/files.js

```diff
@@ -665,119 +521,65 @@ const getDirCID = async (req, res) => {
-    try {
-      // CID is the file CID, parse it from the storagePath
-      const CID = storagePath.split('/').slice(-1).join('')
-      const libs = req.app.get('audiusLibs')
-      await findCIDInNetwork(storagePath, CID, req.logger, libs)
-      return await streamFromFileSystem(req, res, storagePath)
-    } catch (e) {
-      req.logger.error(
-        `Error calling findCIDInNetwork for path ${storagePath}`,
-        e
-      )
-    }
   }
 
+  // Attempt to find and stream CID from other content nodes in the network
   try {
-    // For files not found on disk, attempt to stream from IPFS
-    // Cat 1 byte of CID in ipfs to determine if file exists
-    // If the request takes under 500ms, stream the file from ipfs
-    // else if the request takes over 500ms, throw an error
-    await ipfsSingleByteCat(ipfsPath, req.logContext, 500)
-
-    await new Promise((resolve, reject) => {
-      req.app
-        .get('ipfsAPI')
-        .catReadableStream(ipfsPath)
-        .on('data', (streamData) => {
-          res.write(streamData)
-        })
-        .on('end', () => {
-          res.end()
-          resolve()
-        })
-        .on('error', (e) => {
-          reject(e)
-        })
-    })
+    // CID is the file CID, parse it from the storagePath
+    const CID = storagePath.split('/').slice(-1).join('')
+    const libs = req.app.get('audiusLibs')
+    await findCIDInNetwork(storagePath, CID, req.logger, libs)
+    return await streamFromFileSystem(req, res, storagePath)
   } catch (e) {
+    req.logger.error(
+      `Error calling findCIDInNetwork for path ${storagePath}`,
+      e
+    )
     // Unset the cache-control header so that a bad response is not cached
     res.removeHeader('cache-control')
     return sendResponse(req, res, errorResponseServerError(e.message))
   }
 }
 
 /**
- * Perform IPFS verification with retries
- *
- * Use retries because for some reason IPFS (very rarely) fails due to some non-deterministic error. Seems to be with the verification step; the files are correct.
- * Wait fixed timeout between each retry, hopefully addresses IPFS non-deterministic errors
+ * Verify that content matches its hash using the IPFS deterministic content hashing algorithm.
  * @param {Object} req
  * @param {File[]} resizeResp resizeImage.js response; should be a File[] of resized images
  * @param {string} dirCID the directory CID from `resizeResp`
- * @param {number?} maxRetries the max number of retries for ipfs verification
- * @param {boolean?} enableIPFSAdd flag to enable or disable ipfs daemon add
  */
-const _dirCIDIPFSVerificationWithRetries = async function (
-  req,
-  resizeResp,
-  dirCID,
-  maxRetries = IMAGE_UPLOAD_IPFS_VERIFICATION_RETRY_COUNT,
-  enableIPFSAdd = false
-) {
-  const ipfsAddContent = await _generateIpfsAddContent(resizeResp, dirCID)
+const _verifyContentMatchesHash = async function (req, resizeResp, dirCID) {
```

Comment:
in a followup PR we can prob now just remove this verifyContent logic altogether as there's really nothing to verify

Reply:
@dmanjunath was talking about if that's useful or not in [this comment](https://github.com/AudiusProject/audius-protocol/pull/2772#discussion_r836936573). so you're saying verifying the contents and their hash is not needed in this case (used only in the `/image_upload` route)?

Reply:
lets handle separately, will think about it